### DISCUSSION TITLE
issue on frontend, _video2.default.plugin is not a function

### DISCUSSION THREAD

**Discussion by mrliliang** [mediacms-io/mediacms#1043] [Q&A]
I encountered some issues while debugging the frontend.

When I compile the frontend with command "**npm run dist**" , and copy the static files in folder **frontend/dist/static** to the folder **mediacms/static**，and restart mediacms service,  it can show the video list, but can't play any video. 
When I click a video, the error "**a.default.plugin is not a function**" shows in the browser console.
![image](https://github.com/user-attachments/assets/d8d2ac18-d7d3-48da-b294-82bd3977e5f9)

I also debug the frontend with command "**npm run start**" and open the react pages in the browser, it can display the video list, but can't play any video as well. When I click a video, the error "**_video2.default.plugin is not a function**" is printed in the browser console.
![image](https://github.com/user-attachments/assets/cc177448-fe95-4309-a147-4dfe0b7f795c)

It seems that there are some issues with the javascript file "**mediacms/frontend/src/static/js/components/comments/videojs-markers.js**". 
![image](https://github.com/user-attachments/assets/3a3ef7f4-a4ff-4692-bfaa-a3606e398084)

I printed the object **_video2.default.plugin** in the browser console and found that it was undefined.

Can anyone help me with this issue? 

Thanks!

**Comment by khaphan-github** [comment]
Me too! I can't view the detailed error to troubleshoot it. This error is from the lib folder, but I didn't touch it. :)
